tests/integration/targets/vmware_host_disk_info/tasks/main.yml

```
@@ -0,0 +1,37 @@
+# Test code for the vmware_guest_info module.
+# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright: (c) 2018, James E. King III (@jeking3) <jking@apache.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_attach_host: true
+
+- when: vcsim is not defined
+  block:
+    # Testcase 0001: Get disk info for a host
+    - name: get list of disks from a single host
+      vmware_host_disk_info:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        esxi_hostname: "{{ esxi_hosts[0] }}"
```

```suggestion
        esxi_hostname: "{{ esxi1 }}"
```


This can also use the `esxi1`.  
https://github.com/ansible-collections/community.vmware/blob/358a5ce5175e329b9f1423b0e01680641c363f07/tests/integration/targets/prepare_vmware_tests/vars/common.yml#L6

---

Updated in 18e15942e8ccc85bcb3729134172a32c7bdca295